README.adoc

```diff
@@ -62,7 +62,7 @@ Minishift is an open-source project dedicated to developing and supporting Minis
 The code base is forked from the https://github.com/kubernetes/minikube[Minikube] project.
 
 Contributions, questions, and comments are all welcomed and encouraged!
-Minishift developers hang out on IRC in the #minishift channel on Freenode.
+Join the Minishift team on the #minishift IRC channel on  https://freenode.net/[Freenode] or sign up for the https://lists.minishift.io/admin/lists/minishift.lists.minishift.io[Minishift Mailing List].
```

Comment:
I also like Lala's suggestion but I would rephrase it a little bit to avoid using "join" as the over-arching verb for both actions. The reason for this is that the word "join" sometimes tends to scare potential community members away since it could be implied as "join us a work on stuff", where some people are just users for now who want to ask a question or follow updates.

This is especially true since in the README for example we clearly say that we welcome questions and comments, and not just contributions :)

I would use something similar to this (both in README and INDEX files, we should be consistent):

===

You can reach the Minishift community by:

* Signing up to our [mailing list](https://lists.minishift.io/admin/lists/minishift.lists.minishift.io)
* Joining the #minishift channel on [Freenode IRC](https://freenode.net/)

===

cc @LalatenduMohanty @budhrg @hferentschik @gbraad  - thoughts?